stack.yaml

```diff
@@ -31,15 +31,17 @@ packages:
 # Dependency packages to be pulled from upstream that are not in the resolver
 # (e.g., acme-missiles-0.3)
 extra-deps:
-- git: https://github.com/TorXakis/text-via-sockets.git
-  commit: e3228cd0407ec0d7991a544e154ea2def184fcae
+- git: https://github.com/gijsvcuyck/text-via-sockets.git
+  commit: 8817cb4210103d92ba96aca525ddcd08e1cc96e9
+- MissingH-1.5.0.1
+# Temporary import of deprecated package used by new compiler.
+# Can be removed if txs-compiler module is phased out again for happy based compiler.
+- Unique-0.4.7.9@sha256:7c37e22b7bb9df935c22112e67043177461aedcebe3a958854f6c0e9ecb54083,2155
 
 # Override default flag values for local packages and extra-deps
 flags:
   hexpat:
     bundle: true
-  text:
-    integer-simple: true
```

Comment:
Indeed integer-simple is no longer needed as Haskell is change to use it.

Reply:
When I have time, I will try to remove all mentions of integer-simple and rerun the tests.